### PR TITLE
Update for supporting OpenCV 4.1.1

### PR DESCRIPTION
**Pull request by kumaakh**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds except subtask 10 which always exceeds 50 mins limit on travis, since I ran the builds without any caches.
  Here is the links to the Travis/AppVeyor:
https://ci.appveyor.com/project/kumaakh/hunter/builds/27270564
https://travis-ci.org/kumaakh/hunter/builds/582500465

### CODE REVIEW

File: cmake/projects/OpenCV/hunter.cmake [hunk 1]
```diff
@@ -12,6 +12,21 @@ include(hunter_download)
...
 
 # List of versions here...
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenCV
+    VERSION
+    "4.1.1"
+    URL
+	"https://github.com/kumaakh/hunter-opencv/archive/hunter-4.1.1.zip"
```
Comment (bkotzz) [inline cmake/projects/OpenCV/hunter.cmake]: Let’s get the PR against the cpp-pm fork merged first, and then we can release that and update this link to be the cpp-pm release

**Comment by kumaakh** [comment]
> Nice work!
@headupinclouds mentored me on hunter and also helped me to resolve the last issues. Thanks David.

**Comment by rbsheth** [comment]
Yup, like @bkotzz said, let's get https://github.com/cpp-pm/opencv/pull/1 merged and released first and then we can update this PR before merge. Thanks for your contribution!

**Comment by xsacha** [comment]
Just a note, I have already done OpenCV 4.1.1 in my own hunter fork:
https://github.com/xsacha/opencv/tree/hunter-4.1.1
based on 4.0.0 with minimal changes

**Comment by rbsheth** [comment]
@kumaakh I merged your PR in the OpenCV fork and published a release: https://github.com/cpp-pm/opencv/releases/tag/v4.1.1-p0

Please update this PR to use that release.

**Comment by kumaakh** [comment]
Thanks Rahul, 
I have updated the PR with the correct archive and SHA etc. 
Where is the cpp-pm equivalent of "https://github.com/ingenue/hunter" ?  AKA where should I merge the CI patch ?

**Comment by bkotzz** [comment]
@kumaakh we haven’t moved that over yet. You should be able to use the ingenue one for now. I’m assuming you won’t need to make changes to it

**Comment by kumaakh** [comment]
AKAIK We'll have to merge these changes to pkg.opencv branch of  "https://github.com/ingenue/hunter"  to get clean CI badge

**Comment by bkotzz** [comment]
The main idea is, you need a temporary branch that has the changes you made from this branch, plus the appveyor/Travis changes from the pkg.opencv branch, and push it remotely so that CI runs. You can merge this one into that one, that one into this one, or just checkout those two files into this one. Let me know if you are blocked for any reason

**Comment by kumaakh** [comment]
@bkotzz  thanks for clarifying. 

I submitted a PR to ingenue : https://github.com/ingenue/hunter/pull/392 
And it has kicked off the jobs at [appveyor](https://ci.appveyor.com/project/ingenue/hunter/builds/27469665) and [travis](https://travis-ci.org/ingenue/hunter/builds/586081706)

File: cmake/projects/OpenCV/hunter.cmake [hunk 2]
```diff
@@ -12,6 +12,19 @@ include(hunter_download)
...
 
 # List of versions here...
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenCV
+    VERSION
+    "4.1.1-p0"
+    URL
+	"https://github.com/cpp-pm/opencv/archive/v4.1.1-p0.tar.gz"
```
Comment (rbsheth) [inline cmake/projects/OpenCV/hunter.cmake]: Nit: Can you remove the extra space before the URL?

Comment (kumaakh) [reply cmake/projects/OpenCV/hunter.cmake]: Done with NIT

**Comment by kumaakh** [comment]
Wondering how to avoid the timeouts issue on travis CI. Would a second run work faster due to cacheing ?

**Comment by rbsheth** [comment]
@kumaakh I don't think cache would help, since the OpenCV package didn't complete running but the dependencies should already have been cached. Taking it on faith right now. 🤞

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.216

**Pull request merged**